### PR TITLE
Ci image

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,49 +62,26 @@ jobs:
...
-
-      - name: Update imagemagick
-        if: env.BUILD_TYPE == 'BACKEND'
-        run: |
-          wget https://raw.githubusercontent.com/discourse/discourse_docker/master/image/base/install-imagemagick
-          chmod +x install-imagemagick
-          sudo ./install-imagemagick
-
       - name: Setup redis
         uses: shogo82148/actions-setup-redis@v1
```
Comment (xfalcox) [inline .github/workflows/ci.yml:66]: The image ships with redis already, this should be removed

File: config/unicorn.conf.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # feel free to point this anywhere accessible on the filesystem
 pid (ENV["UNICORN_PID_PATH"] || "#{discourse_path}/tmp/pids/unicorn.pid")
 
-if ENV["RAILS_ENV"] == "development" || !ENV["RAILS_ENV"]
+if ENV["RAILS_ENV"] == "test" || !ENV["RAILS_ENV"]
```
Comment (xfalcox) [inline config/unicorn.conf.rb:28]: Maybe we use `not production`?

**Review by xfalcox — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (CvX) [reply .github/workflows/ci.yml:66]: I know, theoretically the same with postgres.
But I leaving it as is for now intentionally - this setup allows us to easily experiment with different versions. Just pop another number in the version matrix and you're running tests on all declared versions.
Also, `Setup redis` step takes just 1-2s. 😃

File: config/unicorn.conf.rb (hunk 2 above)
Comment (CvX) [reply config/unicorn.conf.rb:28]: This was just for the debugging purposes, but I might leave it in, so I will change it 🙂 👍

**Pull request closed without merging**